vm/vm.go

```diff
@@ -269,8 +277,15 @@ func (vm *VM) Initialize(
 	vm.builder = builder.NewTime(vm)
 	vm.gossiper = txGossiper
 
-	for _, option := range vm.options {
-		if err := option(vm); err != nil {
+	namespacedConfig := make(map[string]json.RawMessage)
+	if err := json.Unmarshal(controllerConfigBytes, &namespacedConfig); err != nil {
+		return fmt.Errorf("failed to unmarshal namespaced config: %w", err)
+	}
+
+	for _, Option := range vm.options {
+		config := namespacedConfig[Option.Namespace]
+		snowCtx.Log.Debug("ctx", zap.Any("name", Option.Namespace), zap.Any("Conf", config))
```

Comment:
Can we remove this log?

We should typically use `zap.String` or `zap.Stringer` instead of `zap.Any` where possible